cluster/src/test/scala/org/apache/spark/sql/store/ColumnUpdateDeleteTest.scala

```diff
@@ -35,11 +49,20 @@ class ColumnUpdateDeleteTest extends ColumnTablesTestBase {
     ColumnUpdateDeleteTest.testBasicDelete(this.snc.snappySession)
   }
 
+  test("SNAP-1925") {
+    ColumnUpdateDeleteTest.testSNAP1925(this.snc.snappySession)
+  }
+
+  test("SNAP-1926") {
+    ColumnUpdateDeleteTest.testSNAP1926(this.snc.snappySession)
+  }
+
   ignore("test update for all types") {
     val session = this.snc.snappySession
     // reduced size to ensure both column table and row buffer have data
     session.conf.set(Property.ColumnBatchSize.name, "100k")
     runAllTypesTest(session)
+    session.conf.unset(Property.ColumnBatchSize.name)
```

Comment:
This is for the case when it is passed the common "this.snc.snappySession" which subsequent tests can re-use. The tests that don't set any explicit ColumnBatchSize intend to use the default one and the unset is for those tests. For tests that need to use a non-default value will set it explicitly in any case.